Makefile

```
@@ -306,7 +306,7 @@ process_openapi: ## Ensure OpenAPI JSON and YAML files are properly formatted
 ##################
 
 .PHONY: trigger_ci
-trigger_ci: ## Trigger the CI pipeline by submitting an empty commit; See https://github.com/pokt-network/pocket/issues/900 for details
+trigger_ci: ## Trigger the CI pipeline by submitting an empty commit; See https://github.com/pokt-network/poktroll/issues/900 for details
```

```suggestion
trigger_ci: ## Trigger the CI pipeline by submitting an empty commit; See https://github.com/pokt-network/pocket/issues/900 for details
```

LOL this must be the only place to go poktroll -> pocket